server/src/main/java/org/opensearch/rest/RestHandler.java

```diff
@@ -171,4 +178,18 @@ public Method getDeprecatedMethod() {
             return deprecatedMethod;
         }
     }
+
+    /**
+     * Replace deprecated route prefix with a new route prefix(_plugins)
+     * @param routes routes
+     * @param prefix new prefix
+     * @param deprecatedPrefix deprecated prefix
+     * @return new list of API routes prefixed with the prefix string
+     * Total number of routes will be the same as number of routes passed.
```

Comment:
I don't think it is necessary to clarify that number of routes will be the same (it was not a suggestion).

Reply:
Got it, thanks Vlad.